src/test/9_Liquidator.t.sol

```diff
@@ -302,6 +302,7 @@ contract LiquidatorTest is DeployArcadiaVaults {
     {
         vm.assume(claimAmount <= openClaim);
         vm.assume(claimAmount <= liquidatorBalance);
+        vm.assume(claimer != liquidityProvider);
```

Comment:
Is this just for simpler balance's in the assertEq's?

Reply:
Yes, otherwise assertEq balance fails